proto/attestation/endorsement.proto

```diff
@@ -44,7 +44,7 @@ message TransparentReleaseEndorsement {
 // <https://www.rfc-editor.org/rfc/rfc9334.html#name-endorsements>
 message Endorsement {
```

Comment:
I think it could be renamed to `Endorsements` (and the corresponding file to `endorsements.proto` too), since `reference_values.proto` is already in plural

Reply:
Singular is also for digest.proto, evidence.proto. I would rather turn reference_values.proto into singular.

Reply:
Fixed singular/plural.